ui/pages/onboarding-flow/secure-your-wallet/index.scss

```diff
@@ -8,6 +8,7 @@
 
   &__details {
     max-width: 550px;
+    padding-right: 10px;
```

Comment:
`padding-inline-end` would be better here for RTL support.

Reply:
@darkwing Fixed in: 58720e7d4f52a70703f7d7f6502b31dcbe7493b3